www/form/app-form.less

```diff
@@ -203,6 +203,28 @@
             //flex-wrap: wrap;
             overflow: auto;
 
+            .cp-form-view-drag {
+                width: 30px;
+                height: 30px;
+                border: none;
+                border-radius: @variables_radius 0px 0px @variables_radius;
+                background-color: @cp_form-bg1;
+                color: @cryptpad_text_col; 
+                @media (max-width:723px) and (min-width:693px), (max-width:553px) and (min-width:524px), (max-width:379px) and (min-width:350px) {
```

Comment:
I was waiting for @davidbenque to review the .less files, but this type of media-query seems very unreliable. If we make a small change to the rest of the UI, this may have to be changed. Additionally, it doesn't cover all cases if you enable the chat and/or the user list:
![image](https://github.com/user-attachments/assets/e20d8711-050a-46ec-9d21-1f22812e2955)